packages/grafana-data/src/transformations/fieldReducer.ts

```diff
@@ -237,6 +238,15 @@ export const fieldReducers = new Registry<FieldReducerInfo>(() => [
     standard: false,
     reduce: (field: Field) => ({ allValues: field.values.toArray() }),
   },
+  {
+    id: ReducerID.allUniqueValues,
+    name: 'All unique values',
+    description: 'Returns an array with all unique values',
+    standard: false,
+    reduce: (field: Field) => ({
+      allUniqueValues: field.values.toArray().filter((n, i) => field.values.toArray().indexOf(n) === i),
```

Comment:
```suggestion
      allUniqueValues: [...new Set(field.values.toArray())],
```
@leeoniya -- can you resolve the best way to do this? :)

Reply:
👍 for the Set / spread strategy